api/hostdb.go

```diff
@@ -0,0 +1,31 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+// ActiveHosts is the struct that pads the response to the hostdb module call
+// "ActiveHosts". The padding is used so that the return value can have an
+// explicit name, which makes adding or removing fields easier in the future.
+type ActiveHosts struct {
+	Entries []modules.HostEntry
+}
```

Comment:
The API should always return objects wrapped in a struct. It makes it easier for us to change things later, and it makes things more generic and predictable.

The big advantage here is you can request the field 'Entries' specifically, so if we change the output, we can change the name and preserve compatibility.